.github/workflows/integration-tests.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       uses: actions/setup-node@v3
       with:
         node-version: ${{ matrix.node-version }}
+    - name: Install higher version of NPM
+      run: npm install -g npm@8.15.0
```

Comment:
Can we have this version of npm defined in just one place so a npm version change only requires it to be done in one place ?

Reply:
Done